src/Greenshot.Editor/Drawing/CropContainer.cs

```diff
@@ -45,13 +70,70 @@ protected override void OnDeserialized(StreamingContext streamingContext)
         }
 
         private void Init()
+        {
+            switch (GetFieldValue(FieldType.CROPMODE))
+            {
+                case CropMode.Horizontal:
+                    {
+                        InitHorizontalCropOutStyle();
+                        break;
+                    }
+                case CropMode.Vertical:
+                    {
+                        InitVerticalCropOutStyle();
+                        break;
+                    }
+                default:
+                    {
+                        InitCropStyle();
```

Comment:
Might it make sense to call the CreateDefaultAdorners directly, without InitCropStyle wrapping it? I couldn't find a second using of it.